tests/unit/insertTestResult.unitTest.ts

```diff
@@ -1411,26 +1408,32 @@ describe("insertTestResult", () => {
 
             testResultsService = new TestResultsService(new MockTestResultsDAO());
 
-            expect.assertions(3);
-            return testResultsService.insertTestResult(testResultWithLecTestTypeWithoutCertNum)
-                .catch((error: { statusCode: any; body: any; }) => {
-                    expect(error).toBeInstanceOf(HTTPError);
-                    expect(error.statusCode).toEqual(400);
-                    expect(error.body).toEqual(ERRORS.NoCertificateNumberOnLec);
+            expect.assertions(7);
+            return testResultsService.insertTestResult(testResultLEC)
+                .then((insertedTestResult: any) => {
+                    expect(insertedTestResult[0].testTypes[0].testTypeId).toEqual("44");
+                    expect(insertedTestResult[0].testTypes[0].certificateNumber).toBe(null);
+                    expect(insertedTestResult[0].testTypes[0].testExpiryDate).toBeTruthy();
+                    expect(insertedTestResult[0].testTypes[0].modType).toBeTruthy();
```

Comment:
Try to  avoid "toBeTruthy" in tests. Errors are truthy. Best to use known values when they aren't dynamically generated. I tend to use Debug to get the expected values.

Reply:
I know the expected values, didn't want to tie this test to the values in the json file. But yes, I will make this test more strict

Reply:
Requested change completed.